opensrp-chw-core/src/test/java/org/smartregister/chw/core/model/VaccineTaskModelTest.java

```diff
@@ -0,0 +1,98 @@
+package org.smartregister.chw.core.model;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.smartregister.domain.Alert;
+import org.smartregister.immunization.domain.Vaccine;
+import org.smartregister.immunization.domain.VaccineWrapper;
+import org.smartregister.immunization.domain.jsonmapping.VaccineGroup;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class VaccineTaskModelTest {
+    private String vaccineGroupName;
+    private VaccineGroup groupMap;
+    private DateTime anchorDate;
+    private List<Alert> alerts;
+    private List<Vaccine> vaccines;
+    private Map<String, Date> receivedVaccines;
+    private List<Map<String, Object>> scheduleList;
+    private ArrayList<VaccineWrapper> notGivenVaccine;
+    private VaccineTaskModel vaccineTaskModel;
+
+    @Before
+    public void setUp(){
+       vaccineGroupName = "some-vaccine-group";
+       groupMap = new VaccineGroup();
+       DateTimeFormatter formatter = DateTimeFormat.forPattern("dd/MM/yyyy");
+       anchorDate = formatter.parseDateTime("01/01/2021");
+       alerts = new ArrayList<>();
+       vaccines = new ArrayList<>();
+       notGivenVaccine = new ArrayList<VaccineWrapper>();
+       vaccineTaskModel = new VaccineTaskModel();
+       scheduleList = new ArrayList<>();
+       receivedVaccines = new HashMap<>();
```

Comment:
It's easily readable if these variables were declared in the required test cases

Reply:
noted